src/actions/actions.js

```diff
@@ -516,6 +516,38 @@ export function exportGist() {
   }
 }
 
+export function saveNotebookServer() {
+  return (dispatch, getState) => {
+    const state = getState()
+    const notebook = {
+      title: state.title,
+      content: exportJsmdToString(state),
+    }
+
+    if (state.userData.accessToken) {
+      fetch('/api/notebook', {
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(notebook),
+        method: 'POST',
+      })
+        .then(response => response.json())
+        .then((json) => {
+          if (json.success) {
+            dispatch(updateAppMessages({
+              message: 'Notebook saved',
+              details: `Notebook created with id ${json.id}`,
+            }))
+          } else {
+            dispatch(updateAppMessages({ message: 'Failed to save notebook' })) // Change msg texts
```

Comment:
This message seems ok to me, at least for now :) Can probably delete the comment